docs/stock/pointandfigure.md

```diff
@@ -0,0 +1,23 @@
+Point and Figure chart
+================
+
+
+A Point and Figure (P&F) chart is a technical analysis tool used in financial markets to track price movements without too much focus on the time. It uses a series of Xs and Os to represent upward and downward price movements, respectively. The Xs indicate price increases, while the Os indicate price decreases. Each symbol is added to the column only when the `boxSize` threshold is crossed. A new column is created only when there is a reversal in the price trend of a predefined amount, known as the `reversalAmount`. This approach filters out minor price fluctuations, focusing only on significant price changes, which helps identify trends, support, and resistance levels. Point and Figure charts are particularly useful for spotting breakouts, trends, and potential entry or exit points in the market.
+
+![pointandfigure.png](pointandfigure.png)
```

Comment:
Wouldn't it be better to add an `iframe` here? - we were aiming for more interactive documentation